packages/components/src/internal/components/assay/AssayPicker.spec.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import {AssayPicker} from '../../..';
+
+import renderer from "react-test-renderer";
+
+describe('AssayPicker', () => {
+    test('AssayPicker', () => {
```

Comment:
are there more jest tests for the Assay picker coming?
a few other cases to cover that I can think of: "Import Assay Design" tab doesn't show when showImport is false, supported file types display changes after specialty assay selection change, component sets the initial tab according to the selectedTab prop.

Reply:
Added more jest testing